packages/expo-screen-orientation/src/ExpoScreenOrientation.ts

```diff
@@ -0,0 +1,3 @@
+import { NativeModulesProxy } from '@unimodules/core';
+
+export default NativeModulesProxy.ExpoScreenOrientation;
```

Comment:
That's how you check for methods availability:
```
if (!ExpoScreenOrientation.getOrientationAsync) {
  throw new UnavailabilityError('ScreenOrientation', 'getOrientationAsync');
}
```
So, what if someone didn't link the module correctly (NativeModulesProxy.ExpoScreenOrientation will be `undefined`)? You should rather do
```suggestion
export default NativeModulesProxy.ExpoScreenOrientation || {};
```